websites/H/HentaiVN/presence.ts

```diff
@@ -0,0 +1,562 @@
+const presence = new Presence({
+		clientId: "942782020055089192"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+let page = "1",
+	keyword,
+	namekey,
+	doukey,
+	charkey,
+	keystr,
+	largebox;
+
+presence.on("UpdateData", async () => {
+	const inURL = new URLSearchParams(document.location.search),
+		inPath = document.location.pathname,
+		inQuery = document.location.search,
+		select = document.querySelector,
+		selectAll = document.querySelectorAll,
```

Comment:
```suggestion
	const { pathname, search} = document.location
		searchParams = new URLSearchParams(search),
```

Reply:
done, thanks

Reply:
No, use my suggestion and don't repeat `document.location.pathname` and use `pathname` instead.. and change the variable name of the SearchParmas getter

Reply:
This...

Reply:
> No, use my suggestion and don't repeat `document.location.pathname` and use `pathname` instead.. and change the variable name of the SearchParmas getter

thanks!